### ISSUE TITLE
Cannot separate cards

### ISSUE THREAD

**Issue by fuen0015** [fuen0015/animal-rescue-website#1]
Hello thomas,
I am trying to make cards for the animal rescue website assignment but I am having trouble separating my cards when they get to large screens. What should I do?
@thomasjbradley

**Comment by fuen0015** [comment]
I actually fixed it but now a new problem, I cannot remove the numbers from my lists, I put list-style-type: none; in my css but it was still there.

**Comment by thomasjbradley** [comment]
Try adding the class `.list-group`

**Comment by fuen0015** [comment]
I changed everything but now my cards are back to being stuck together any suggestions?

**Comment by fuen0015** [comment]
Also my icons are huge and I need them to be small

**Comment by thomasjbradley** [comment]
The stuck cards are on the “Rescues” page? I don’t see them.

**Comment by thomasjbradley** [comment]
*Re: icons—*

They have the `.i-192` size on them, which is pretty big. Try a smaller size.

**Comment by fuen0015** [comment]
Oh sorry I meant on my index.html

**Comment by thomasjbradley** [comment]
I’m sorry. I don’t see what cards you are talking about on the homepage. The ones with the icons?

**Comment by fuen0015** [comment]
Let me rephrase my question. My social media icons on footer in my index.html are too large, how can I reduce the size

**Comment by thomasjbradley** [comment]
Um. Have the social icons been committed? I can’t seem to find them.

**Comment by fuen0015** [comment]
I think you should see it now, I tried changing there size in css but I don't that this is the correct way to do it

**Comment by thomasjbradley** [comment]
The `<svg>` tags are not inside the `<i>` tag. The `<i>` tag is controlling the size, but they’re completely empty.
